Automattic/wp-calypso
# Global sidebar mobile - add site card to top bar

Related to # https://github.com/Automattic/dotcom-forge/issues/5752

## Proposed Changes

* Shows the site card in the global sites topbar for mobile
* removes the site card from the sidebar in this case and adds a "my home" link
* updates global masterbar header to be the same color as global sidebar.
* TODO - support the site badges in the card in this new location. Either in this PR or follow up...


Before:
<img width="760" alt="Screenshot 2024-02-26 at 2 11 37 PM" src="https://github.com/Automattic/wp-calypso/assets/28742426/410493a8-1ed7-4857-b12f-149ad524e1f0">


After:
<img width="748" alt="Screenshot 2024-02-26 at 1 25 14 PM" src="https://github.com/Automattic/wp-calypso/assets/28742426/0a6fc59c-8405-4649-a818-2e2f6fdcd9fd">


## Testing Instructions



* Run this branch of calypso.
* Visit global site view for an atomic site.
* Use a viewport width thinner than 782px.
* Verify the top bar is the same color as the global sidebar
* verify the top bar now contains the Site card in place of the "W" logo.
* verify this loads my home when clicked.
* open the sidebar, verify there is no site card. Verify there is a "my home" menu item.
* visit global site view in desktop width, there should be no "my home" item and the site card should be there.
* visit global view in mobile width, verify the "W" logo is still in the top bar.

## Pre-merge Checklist
